src/main/scala/mesosphere/marathon/MarathonSchedulerActor.scala

```diff
@@ -300,7 +303,7 @@ object MarathonSchedulerActor {
 
   case object ReconcileHealthChecks
 
-  case class ScaleApp(appId: PathId) extends Command {
+  case class ScaleApp(appId: PathId, force: Boolean = false) extends Command {
```

Comment:
Also, since there are only two use-sites total I'm not sure the default value buys us much.  On the other hand, it makes the use site where the default is not overridden harder to understand.